app.js

```diff
@@ -25,15 +26,17 @@ app.get('/', function get(req, res) {
 });
 
 app.get('/responses', function get(req, res) {
-return smartSurveyAPICall.getData(
-    config.apiToken,
-    config.apiTokenSecret,
-    config.surveyID)
-  .then(results => res.json(results))
-  // This is a placeholder for error logging. In the future, the aim
-  // is to print a friendly message to the user with the stacktrace
-  // eslint-disable-next-line no-console
-  .catch(error => console.log(error));
+  const options = {
```

Comment:
Although I would probably pass in the individual bits of config separately to avoid tightly coupling with the config module.